frontend/src/js/tooltip/AdditionalInfoHoverable.tsx

```diff
@@ -15,36 +16,39 @@ const Root = styled("div")`
 // Allowlist the data we pass (especially: don't pass all children)
 const getAdditionalInfos = (
   node: ConceptT,
-  parent?: ConceptT,
+  root?: ConceptT,
 ): AdditionalInfosType => ({
   label: node.label,
   description: node.description,
-  isFolder: !!node.children && node.children.length > 0,
   matchingEntries: node.matchingEntries,
   matchingEntities: node.matchingEntities,
   dateRange: node.dateRange,
   infos: node.additionalInfos,
-  parent: parent,
+  icon: getNodeIcon(node, {
+    isStructNode: !root?.detailsAvailable,
+  }),
+  rootLabel: root?.label,
+  rootIcon: root ? getNodeIcon(root) : undefined,
```

Comment:
Computing the icon already in the action payload, allows to remove `isFolder` and the only thing left that's required from `root` is its `label` then. That avoids passing a whole node (potentially a very large object including all children, not so great with redux state)